watertap/unit_models/tests/test_anaerobic_digestor.py

```diff
@@ -163,269 +111,60 @@ def adm(self):
-                "Heat Duty": adm.fs.unit.heat_duty[0],
-            }
+        self.unit_model_block = m.fs.unit
+        self.unit_statistics = {
+            "number_variables": 203,
+            "number_total_constraints": 171,
+            "number_unused_variables": 0,
         }
-
-    @pytest.mark.unit
-    def test_report(self, adm):
-        adm.fs.unit.report()
+        self.unit_solutions[m.fs.unit.liquid_outlet.pressure[0]] = 101325
+        self.unit_solutions[m.fs.unit.liquid_outlet.temperature[0]] = 308.15
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_I"]] = 0.33
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_aa"]] = 0.0053
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_ac"]] = 0.2
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_bu"]] = 0.013
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_ch4"]] = 0.055
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_fa"]] = 0.099
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_h2"]] = 2.3e-07
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_pro"]] = 0.016
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_su"]] = 0.012
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_va"]] = 0.012
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_I"]] = 25.6
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_aa"]] = 1.18
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_ac"]] = 0.76
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_c"]] = 0.31
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_c4"]] = 0.43
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_ch"]] = 0.028
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_fa"]] = 0.24
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_h2"]] = 0.32
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_li"]] = 0.029
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_pr"]] = 0.10
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_pro"]] = 0.14
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "X_su"]] = 0.42
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_IC"]] = 1.8
+        self.unit_solutions[m.fs.unit.liquid_outlet.conc_mass_comp[0, "S_IN"]] = 1.82
+        self.unit_solutions[m.fs.unit.liquid_outlet.anions[0]] = 0.02
+        self.unit_solutions[m.fs.unit.liquid_outlet.cations[0]] = 0.04
+        self.unit_solutions[m.fs.unit.vapor_outlet.pressure[0]] = 101325
+        self.unit_solutions[m.fs.unit.vapor_outlet.temperature[0]] = 298.15
+        self.unit_solutions[m.fs.unit.vapor_outlet.flow_vol[0]] = 1
```

Comment:
@bknueven I haven't been able to figure out why, but something about the latest commit changed the expected values for a lot of these variables, and I don't think they're accurate anymore. For example, the `m.fs.unit.vapor_outlet.temperature` should be 308.15, not 298.15. It looks like pressure, temperature and volumetric flow are getting set to default values...

Reply:
Maybe try un-doing this suggestion first? https://github.com/watertap-org/watertap/pull/1277#discussion_r1469970321

If that works let me know...

Reply:
Okay. Ignore my earlier comment and leave a note about it where you create that block.

There's surely a good Pyomo explanation but I cannot think of the cause off the top of my head. We should probably investigate further, but that's not needed for this PR.